common/views/themes/default.ts

```diff
@@ -185,4 +186,32 @@ const theme = {
   makeSpacePropertyValues,
 };
 
+const GlobalStyle = createGlobalStyle`
+  ${css`
+    .block {
+      display: block;
+    }
+    .none {
+      display: none;
+    }
+  `}
+  ${css`
+    ${Object.keys(themeValues.sizes).map(
+      size => css`
+        .${size}-none {
```

Comment:
Could we go with `-show` and `-hide`?

Reply:
The reason I have been avoiding that is `hide` is quite ambivalent, could be `visibility: hidden` or similar.

I have avoided using `-show` as you might want to use a different `display: ` value later, so this should help this scale.

I could prefix it with `{size}-display-show` or something?

Reply:
yeah, I was mostly thinking about something to make it less ambiguous – `none` isn't obviously about `display`.